Makefile

```diff
@@ -13,6 +13,18 @@ MASTER_IMAGE = quay.io/dynatrace/dynatrace-operator:snapshot
 BRANCH_IMAGE = quay.io/dynatrace/dynatrace-operator:snapshot-$(shell git branch --show-current | sed "s/[^a-zA-Z0-9_-]/-/g")
 OLM_IMAGE ?= registry.connect.redhat.com/dynatrace/dynatrace-operator:v${VERSION}
 
+KUBERNETES_OTHERS_YAML=config/deploy/kubernetes/kubernetes-others.yaml
```

Comment:
I would suggest using a tmp file to do the file manipulations which can be discarded afterwards.
There is no need as far as I can see to keep a file with a specific name

Reply:
These intermediate results are git-ignored and it allows developer to understand the intermediate results. I'd prefer to leave it as it is now.

Reply:
I think @mjgrzybek is right here, they are git-ignored and now you can actually see the results in each and check, in which file an error happened in case